wordpress/wp-content/plugins/cds-base/classes/Modules/Notify/NotifySettings.php

```diff
@@ -117,7 +116,7 @@ function ($input) {
 
         add_settings_section(
             'notify_api_settings_setting_section', // id
-            __('GC Lists Settings', 'cds-snc'), // title
+            __('Notify API Settings', 'cds-snc'), // title
```

Comment:
Now that the settings are under `GC Lists` was thinking this should be GC Lists as a heading but yes they are Notify API --- what's in Figma?